app/views/cities/forbes_show.html.erb

```diff
@@ -30,11 +30,9 @@
   	<% end %>
     <% if @city.unapproved? %>
     <% else %>
-      <div class='sharing-info'>
-        <%= render partial: 'shared/sharing_widget', locals: {city: @city} %>
-      </div>
     <% end %>
     -->
+    <%= render partial: 'shared/sharing_widget', locals: {city: @city} %>
```

Comment:
Not using `city` local (yet).

Can add a custom fallback text here (e.g. for Twitter).